src/app/lists/EditModal.tsx

```diff
@@ -21,6 +21,11 @@ interface Props {
 // null => add, undefined => modal closed
 export type EditModalState = Schema["lists"] | null | undefined;
 
+type AddEditListErrorTypes = "failedToAddListItem" | "failedToEditListItem";
+type AddEditListReturn = {
+    error: { type: AddEditListErrorTypes; logId: string } | null;
+};
```

Comment:
I feel like it'd make sense to have separate types for add and edit operations. The caller of addListItem shouldn't be expecting `"failedToEditListItem"` as a possible error type right?